tpp-app/src/services/utils/helpers.js

```diff
@@ -92,23 +96,25 @@ export const getCalendarByYear = async (year) => {
   return calendars;
 }
 
+
 /**
  * Retrieves the user's symptom data for the given date from the calendar.
 * @param {Object} calendar The object containing the symptoms for this year, last year, and next year.
- * @param {Number} day number (First day = 0)
+ * @param {Number} day number (First day = 1)
  * @param {Number} month number (January = 1)
  * @param {Number} year number
  */
 export const getSymptomsFromCalendar = async (calendar, day, month, year) => {
```

Comment:
just heads up @roylin2000 pointed out this should be a synchronous function since it doesn't need to be async

Reply:
nvm lol i didn't see u call it anywhere